### PR TITLE
21-0845 - Fix hint-text screen-reader bug

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Fix Third-Party Authorization (21-0845) a11y screen-reader bug where hint-text `'ui-description'`s were not read out.
- Team: Platform VA Product Forms
- \[Flipper not implemented yet; disabled in Prod via content-build's app-registry setting\]

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#64780

## Testing done

- Local manual-UI and unit tests all successful.  \[E2E-spec's CI-skipped.  Still flaky.\]

## What areas of the site does it impact?

This form ONLY.

### CODE REVIEW

File: src/applications/simple-forms/21-0845/pages/organizationReps.js [hunk 1]
```diff
@@ -6,11 +6,15 @@ import OrgRepsViewField from '../components/OrgRepsViewField';
 export default {
   uiSchema: {
     'ui:title': (
-      <h3 className="custom-header">Organization’s representatives</h3>
+      <>
+        <p className="vads-u-font-size--lg">Organization’s representatives</p>
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-0845/pages/organizationReps.js:10]: wouldn't we want to keep this in at h3? and just style it how we want?

File: src/applications/simple-forms/21-0845/pages/authorizerType.js [hunk 2]
```diff
@@ -9,11 +9,13 @@ export default {
     authorizerType: {
       'ui:title': (
         <>
-          <h2 className="custom-header authorizer-type vads-u-font-size--h3">
+          <span className="custom-header authorizer-type vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h3">
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-0845/pages/authorizerType.js:12]: from a content structure point of view this feels like it should be an h2. what's the reason for us not using h2 here?

**Review by rhasselle-oddball — COMMENTED** [review]
looks good, but a little confused on some of the h2/h3 changes. maybe we can discuss offline

File: src/applications/simple-forms/21-0845/pages/organizationReps.js (hunk 1 above)
Comment (tlei123) [reply src/applications/simple-forms/21-0845/pages/organizationReps.js:10]: It's the Review page where using <h> tags gets you into a11y-hierarchy issues; not on the form-page itself.  I just found out the hard way.

File: src/applications/simple-forms/21-0845/pages/authorizerType.js (hunk 2 above)
Comment (tlei123) [reply src/applications/simple-forms/21-0845/pages/authorizerType.js:12]: Same as above, <h> tags in <legends> get pulled as-is into Review-page and generates a11y headers-hierarchy bugs.

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**